### PR TITLE
Setup Sentry

### PR DESCRIPTION
**Pull request by vademo**
NEB-32

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,3 +4,6 @@ PORT=
 # DEFAULT: /api/v1
 BASE_PATH=
 
+# Sentry log https://*****@sentry.io/****
+#set to false for no loggin
```
Comment (hoetmaaiers) [inline .env.example]: Space before the comment pls.

File: src/index.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import dotenvSafe = require('dotenv-safe');
+import raven = require('raven');
```
Comment (hoetmaaiers) [inline src/index.ts]: This is weird syntax, both line 1 & 2. I suggest to always use module import like this:

```
import dotenvSafe from 'dotenv-safe';
import Raven from 'raven';
```

**Review by hoetmaaiers — COMMENTED** [review]
Is a Sentry project available to test and log crashes for this boilerplate? If not I can create a project.

**Comment by vademo** [comment]
@hoetmaaiers Tested with my own Senty account. Havent got access to the iCapps Sentry yet

File: src/index.ts [hunk 3]
```diff
@@ -11,6 +13,10 @@ import models from './models';
 import Example from './models/Example';
 
 const config = {};
+// configure loggin
+if (process.env.SENTRY_DSN !== 'false') {
```
Comment (samover) [inline src/index.ts]: I suggest placing this in `config/sentry.config` 
Any opinions on this? I would like to see all that even smells like configuration put into separate files :-)

Comment (hoetmaaiers) [reply src/index.ts]: Would you export a `Raven` instance from the config file then? Or what is the idea?

**Review by hoetmaaiers — DISMISSED** [review]

**Review by samover — APPROVED** [review]

**Pull request merged**